clang/Type.d

```diff
@@ -12,6 +12,14 @@ import clang.c.Index;
 import clang.Cursor;
 import clang.Util;
 
+class TypeIsNotAnArrayError : object.Error
+{
```

Comment:
No. But on the other hand, can we use an assert instead? It's a logical error to call `numDimensions` on something that isn't an array?